buch/src/components/LoginBox/index.scss

```diff
@@ -0,0 +1,39 @@
+@import "../../style/vars.scss";
+
+.loginbox{
+    border: 1px;
+    border-style: solid;
+    border-color: #eee;
+    padding: 5px;
+    background-color: $background-secondary;
+}
+  
+.text-input{
+    margin-top: 2px;
+    margin-bottom: 2px;
+    margin-left: 5px;
+    margin-right: 5px;
+}
+
+.loginbox-buttons{
+    margin: 5px;
+}
+
+.loginbox-button{
```

Comment:
Change this to use the scss syntax instead of multiple different selector. So instead of individual selectors for :hover and :active, put the changes for these selectors inside of the main .loginbox-button with &:hover {} and &:active {}